src/index.ts

```diff
@@ -53,6 +54,45 @@ export default class FiatConnectClient implements FiatConectApiClient {
     }
   }
 
+  /**
+   * https://en.wikipedia.org/wiki/Network_Time_Protocol#Clock_synchronization_algorithm
+   *
+   * Convenience method to calculate the difference between server and client clocks.
+   * Returns the difference between the client and server clocks as a number of milliseconds.
+   * Positive values mean the server's clock is ahead of the client's.
+   */
+  async getClockDiff(): Promise<Result<number, ErrorResponse>> {
+    const t0 = Date.now()
+    const clockResponse = await this.getClock()
+    const t3 = Date.now()
+
+    if (!clockResponse.ok) {
+      return clockResponse
+    }
+
+    const t1 = new Date(clockResponse.val.time).getTime()
+    // We can assume that t1 and t2 are sufficiently close to each other
+    return Ok((t1 - t0 + t1 - t3) / 2)
```

Comment:
Should we add a Math.floor() or something in here so we don't return fractions of milliseconds?

Reply:
➕

Reply:
Good idea! Added.